Update Glossary-GeneralBitcoin.md (#718)
zkSNACKs/WasabiDoc: docs/glossary/Glossary-GeneralBitcoin.md

Read more: [Bitcoin addresses](/using-wasabi/Receive.md#bitcoin-public-keys-and-addresses)
:::

:::details
### Bitcoin

> Bitcoin is an innovative payment network and a new kind of money. Bitcoin is a cryptocurrency, a form of electronic cash. It is a decentralized digital currency without a central bank or single administrator that can be sent from user to user on the peer-to-peer bitcoin blockchain network without the need for intermediaries.

Reviewer: ```suggestion
Bitcoin is an innovative payment network and a new kind of money.
Bitcoin is a cryptocurrency, a form of electronic cash.
It is a decentralized digital currency without a central bank or single administrator that can be sent from user to user on the peer-to-peer bitcoin blockchain network without the need for intermediaries.
```

Read more: [Bitcoin addresses](/using-wasabi/Receive.md#bitcoin-public-keys-and-addresses)
:::

:::details
### Bitcoin

> Bitcoin is an innovative payment network and a new kind of money.
> Bitcoin is a cryptocurrency, a form of electronic cash.

Reviewer 2: ```suggestion
Bitcoin is a peer-to-peer electronic cash system.
An innovative payment network and a new kind of money.
```

In a typical payment channel, only two transactions are added to the blockchain but an unlimited or nearly unlimited number of payments can be made between the participants.
:::

:::details
### Plausible deniability

> Plausible deniability is the ability to deny knowledge of or responsibility for any damnable actions committed by others in an organizational hierarchy because of a lack of evidence that can confirm their participation, even if they were personally involved in or at least willfully ignorant of the actions. 

Reviewer: ```suggestion
Plausible deniability is the ability to deny knowledge of or responsibility for any damnable actions committed by others in an organizational hierarchy because of a lack of evidence that can confirm their participation, even if they were personally involved in or at least willfully ignorant of the actions.
```
